### PR TITLE
Create report handling manual in CoC

### PR DESCRIPTION
**Pull request by malvikasharan**
## Summary

Following discussion in https://github.com/the-turing-way/the-turing-way/issues/3229 -- creating this PR to add a Report Handling Manuel.

The text is reused from another CoC where the process was reviewed by some of The Turing Way members: https://github.com/alan-turing-institute/open-research-community-management/blob/main/CODE_OF_CONDUCT.md

## Reviewers should focus on

- If the Report Handling is clearly described
- Any concerns they have
- Anything missing
- If the scope of CoC can be clarified (see the scope section in the document to this PR)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+The committee will determine the cause and consequences that violated this Code of Conduct and take further actions.
+
+### 4.1 If the breach is considered within the scope of this Code of Conduct
+
+The committee will reach out to the reporter to ask for further information if the 
+the submitted report is not complete or needs some additional context.
+The committee will also ensure that the person impacted or harmed by the incident report
+is safe and identifies how they can be further supported.
+
+**4.1.1** When the committee decides to meet the reporter, these questions will be used to guide the meeting:
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:136]:
```suggestion
**4.1.1** When the committee decides to meet the reportee, these questions will be used to guide the meeting:
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+Due to the severity of the incident reports that go beyond norm-setting in *The Turing Way*, an appropriate 
+(external) authority such as the reportee's employee/line manager will be identified
+who can respond to the incident report with the appropriate action.
+The report will be passed to the identified authority with an agreement to receive an update from their enforcement process.
+
+The resolution or decision made by the external authority may or may not have an impact on the internal resolution.
+
+A transparency report will be drafted by the committee and shared with the community where appropriate.
+
+### 4.3 If the reportee is an employee of the Turing
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:175]:
```suggestion
### 4.3 If the breach is considered outside the scope of this Code of Conduct and the reportee is an employee of the Turing

The Code of Conduct should be considered a norm-setting mechanism for all community members, including the Turing employees.


```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+
+The committee may follow up with reportee or reporter after this discussion 
+with any additional questions to help them reach a resolution and must note feedback for the record.
+
+Mediating interaction or communication between reporter and reportee 
+is not in the scope of this Code of Conduct.
+
+### 4.2 If the breach is considered outside the scope of this Code of Conduct
+
+The Code of Conduct should be considered a norm-setting mechanism for all community members, including the Turing employees.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:158]:
```suggestion
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -107,28 +107,119 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+The resolution or decision made by the external authority may or may not have an impact on the internal resolution.
+
+A transparency report will be drafted by the committee and shared with the community where appropriate.
+
+### 4.3 If the breach is considered outside the scope of this Code of Conduct and the reportee is an employee of the Turing
+
+The Code of Conduct should be considered a norm-setting mechanism for all community members, including the Turing employees.
+
+
+
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:179]:
```suggestion
```

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+A transparency report will be drafted by the committee and shared with the community where appropriate.
+
+### 4.3 If the breach is considered outside the scope of this Code of Conduct and the reportee is an employee of the Turing
+
+The Code of Conduct should be considered a norm-setting mechanism for all community members, including the Turing employees.
+For all incident reports that are considered beyond the scope of this Code of Conduct, **the Turing Institute's employment policies will apply**.
+
+The committee will **escalate the issue to Kirstie Whitaker** (if not already involved), 
+who is ** project lead and Director of the Tools, Practices and Systems Research Programme at the Turing.
+She will share the report and committee's recommendations with the line manager of the reporter,
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:181]:
```suggestion
She will share the report and committee's recommendations with the line manager of the reportee, 
```

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+## 3.6 Terminology Used in Report Handling and Enforcement Manuals
+
+* Reporter: Person reporting an incident.
+* Reportee: Person being reported.
+
+## 4 Report Handling Manual
+
+After receiving a report, the committee will review the incident report and follow the policies 
+described in this Report Handling Manual.
+The committee will determine the cause and consequences that violated this Code of Conduct and take further actions.
```
Comment (gedankenstuecke) [inline CODE_OF_CONDUCT.md:127]: _the cause and consequences that violated_ reads slightly strange to me, is this meant to indicate a difference between immediate cause (e.g. "consequences") & "root cause" somehow?

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+
+### 4.1 If the breach is considered within the scope of this Code of Conduct
```
Comment (gedankenstuecke) [inline CODE_OF_CONDUCT.md:129]: This might be nitpicky, but "breach" here somewhat pre-supposes that the reported incident was a breach? But I guess a report can be within scope but ultimately "not a breach"?

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+- What has been the hardest thing for you?
+- What do you think needs to happen to make things right?
+
+The committee may follow up with reportee or reporter after this discussion 
+with any additional questions to help them reach a resolution and must note feedback for the record.
+
+Mediating interaction or communication between reporter and reportee 
+is not in the scope of this Code of Conduct.
+
+### 4.2 If the breach is considered outside the scope of this Code of Conduct
```
Comment (gedankenstuecke) [inline CODE_OF_CONDUCT.md:156]: I was about to post the same comment on using "breach" here, but now, after reading this section it makes me wonder if the goal of the "report handling manual" is to cover these cases where "we find there was no breach of the CoC"?

File: CODE_OF_CONDUCT.md (hunk 7 above)
Comment (gedankenstuecke) [reply CODE_OF_CONDUCT.md:129]: And I'm not sure if I missed it somewhere, but I could not find a description of what would be considered within/outside the scope of the CoC?

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md:127]: Good point. Let me rephrase that.

File: CODE_OF_CONDUCT.md (hunk 7 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md:129]: Let me rephrase to make it clear that this section applies only if a breach has been identified and is also considered severe enough to escalate.

There is no scope section, and i agree that there should be. That's one of my original ask for this PR. Your response is definitely nudging to having an explicit scope section.

File: CODE_OF_CONDUCT.md (hunk 8 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md:156]: True. That can make it a bit more open to the fact that not all reports will be about breach of CoC.

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:127]:
```suggestion
Using the report handling and enforcement process described in this document, the committee will handle the Code of Conduct and take further actions.
```

File: CODE_OF_CONDUCT.md (hunk 7 above)
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:129]:
```suggestion
### 4.1 If the breach has been identified and is considered within the scope of this Code of Conduct
```

File: CODE_OF_CONDUCT.md [hunk 9]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+* Reporter: Person reporting an incident.
+* Reportee: Person being reported.
+
+## 4 Report Handling Manual
+
+After receiving a report, the committee will review the incident report and follow the policies 
+described in this Report Handling Manual.
+Using the report handling and enforcement process described in this document, the committee will handle the Code of Conduct and take further actions.
+
+### 4.1 If the breach has been identified and is considered within the scope of this Code of Conduct
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:129]:
```suggestion
### 4.1 If the report is considered within the scope of this Code of Conduct
```

File: CODE_OF_CONDUCT.md [hunk 10]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+- What has been the hardest thing for you?
+- What do you think needs to happen to make things right?
+
+The committee may follow up with reportee or reporter after this discussion 
+with any additional questions to help them reach a resolution and must note feedback for the record.
+
+Mediating interaction or communication between reporter and reportee 
+is not in the scope of this Code of Conduct.
+
+### 4.2 If the breach is considered outside the scope of this Code of Conduct
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:156]:
```suggestion
### 4.2 If a breach has been identified within the scope of this Code of Conduct
```

File: CODE_OF_CONDUCT.md [hunk 11]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
 
 If you believe someone is in physical danger, please contact the appropriate emergency responders.
 
-## 3.4 Code of Conduct Enforcement
+## 3.4 Report handling 
+
+A detailed report handling policy is available in the Report Handling Manual.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:112]:
```suggestion
A detailed report handling policy is available in the Report Handling Manual in section 4.
```

File: CODE_OF_CONDUCT.md [hunk 12]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+
+## 3.5 Code of Conduct Enforcement
 
 A detailed enforcement policy is available in the Enforcement Manual.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:116]:
```suggestion
A detailed enforcement policy is available in the Enforcement Manual in section 5.
```

File: CODE_OF_CONDUCT.md [hunk 13]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
 
-# 4 Enforcement Manual
+## 3.6 Terminology Used in Report Handling and Enforcement Manuals
+
+* Reporter: Person reporting an incident.
+* Reportee: Person being reported.
+
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:122]:
```suggestion

## 3.7 Scope of this Code of Conduct

This Code of Conduct applies to all online and in-person events, and meetings conducted in *The Turing Way* community spaces (online/in-person), as well as all text-based synchronous and asynchronous interactions as described below.

- The Turing Way organised meetings, such as Collaboration Cafés, Book Dashes, Fireside Chats, community calls,  project meetings (both openly hosted meetings for the community and closed meetings for the team only),  and similar events. 
- The Turing Way team and community leaders such as a sub-project or Working Group member, who attend an event or community spaces while representing The Turing Way.
- Community members who take formal roles in *The Turing Way*, internships or in formal mentorship roles such as through Outreachy, Google Summer of Code, Google Season of Doc and similar time-bound activities.
- Communication and community engagement platforms, such as project repositories on GitHub (issues, PRs or other contributions), Slack Workspace and workshops or training events, and social media platforms such as X/Twitter, Mastodon and LinkedIn.

For events where *The Turing Way* is not the main organiser and if a separate Code of Conduct applies, it will be stated clearly.
```

File: CODE_OF_CONDUCT.md (hunk 9 above)
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:129]:
```suggestion
### 4.1 If the Report is considered within the scope of this Code of Conduct
```

File: CODE_OF_CONDUCT.md [hunk 14]
```diff
@@ -107,28 +107,116 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+The committee may follow up with reportee or reporter after this discussion 
+with any additional questions to help them reach a resolution and must note feedback for the record.
+
+Mediating interaction or communication between reporter and reportee 
+is not in the scope of this Code of Conduct.
+
+### 4.2 If the breach is considered outside the scope of this Code of Conduct
+
+The Code of Conduct committee will assess if the report is outside the scope of *The Turing Way* Code of Conduct enforcement process.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:158]:
```suggestion
```

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md]: Made changes.

File: CODE_OF_CONDUCT.md (hunk 7 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md]: I have added a scope section.

File: CODE_OF_CONDUCT.md [hunk 15]
```diff
@@ -107,28 +107,115 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+Due to the severity of the incident reports that go beyond norm-setting in *The Turing Way*, an appropriate 
+(external) authority such as the reportee's employee/line manager will be identified
+who can respond to the incident report with the appropriate action.
+The report will be passed to the identified authority with an agreement to receive an update from their enforcement process.
+
+The resolution or decision made by the external authority may or may not have an impact on the internal resolution.
+
+A transparency report will be drafted by the committee and shared with the community where appropriate.
+
+### 4.3 If the breach is considered outside the scope of this Code of Conduct and the reportee is an employee of the Turing
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:173]:
```suggestion
### 4.3 If the report is considered outside the scope of this Code of Conduct and the reportee is an employee of the Turing
```

File: CODE_OF_CONDUCT.md [hunk 16]
```diff
@@ -107,28 +107,115 @@ You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@t
...
+
+A transparency report will be drafted by the committee and shared with the community where appropriate.
+
+### 4.3 If the report is considered outside the scope of this Code of Conduct and the reportee is an employee of the Turing
+
+The Code of Conduct should be considered a norm-setting mechanism for all community members, including the Turing employees.
+For all incident reports that are considered beyond the scope of this Code of Conduct, **the Turing Institute's employment policies will apply**.
+
+The committee will **escalate the issue to Kirstie Whitaker** (if not already involved), 
+who is ** project lead and Director of the Tools, Practices and Systems Research Programme at the Turing.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:179]:
```suggestion
who is the project lead and Director of the Tools, Practices and Systems Research Programme at the Turing.
```
